conf/storm.yaml.example

```diff
@@ -19,7 +19,7 @@
 #     - "server1"
 #     - "server2"
 # 
-# nimbus.host: "nimbus"
+# nimbus.seeds: ["localhost"]
```

Comment:
Can you instead use values that will be valid in a single node setup? With current values anyone running storm locally will be forced to update this value.

nimbus.seeds:["localhost","127.0.0.1"]